Dockerfile

```diff
@@ -8,49 +8,39 @@ ENV PYTHONUNBUFFERED 1
 # Set working directory
 WORKDIR /app
 
-RUN apt-get -qq update
-RUN DEBIAN_FRONTEND="noninteractive" apt-get install -qq -y --fix-missing \
-    python3-dev \
-    python3-pip \
-    curl \
-    git
-
-RUN curl -L -o /tmp/dlite.deb https://github.com/SINTEF/dlite/releases/download/0.3.1/dlite-0.3.1-x86_64.deb
-
-RUN apt-get install -y -f /tmp/dlite.deb \
-  && rm -rf /var/lib/apt/lists/*
-
-# Install requirements
-COPY ./requirements.txt .
-RUN pip install -q --no-cache-dir --trusted-host pypi.org --trusted-host files.pythonhosted.org --upgrade pip
-RUN pip install -q --trusted-host pypi.org --trusted-host files.pythonhosted.org --trusted-host github.com -r requirements.txt
-ENV DLITE_ROOT=/usr
-ENV DLITE_STORAGES=/app/entities/*.json
-ENV PYTHONPATH=/usr/lib64/python3.9/site-packages
-RUN mkdir -p /app/entities
+# Copy core parts and install requirements
+COPY app app/
+COPY requirements.txt LICENSE README.md asgi.py ./
+RUN apt-get update \
+  && apt-get -y install --fix-broken --fix-missing --no-install-recommends git \
+  && apt-get purge -y --auto-remove \
+  && rm -rf /var/lib/apt/lists/* \
+  && pip install --no-cache-dir --trusted-host pypi.org --trusted-host files.pythonhosted.org --upgrade pip setuptools wheel \
+  && pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org --trusted-host github.com -r requirements.txt
 
 ################# DEVELOPMENT ####################################
 FROM base as development
-COPY . .
 
-# Run static security check and linters
-RUN pip install -q --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements_dev.txt
+# Copy development parts
+COPY tests tests/
+COPY .git .git/
+COPY requirements_dev.txt .pre-commit-config.yaml .pylintrc ./
+
+# Run static security check, linters, and pytest with code coverage
+RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements_dev.txt \
```

Comment:
I disagree. It's better to combine as many tasks into a single RUN command as possible to avoid creating too many image layers. Each separate command creates a separate layer when building the image.